crates/bevy_render/src/view/window.rs

```diff
@@ -270,5 +305,48 @@ pub fn prepare_windows(
             }
         };
         window.swap_chain_texture_format = Some(surface_data.format);
+
+        if window.screenshot_func.is_some() {
+            let texture = render_device.create_texture(&wgpu::TextureDescriptor {
+                label: None,
```

Comment:
I'd recommend adding labels here and the rest of the code. It makes things way easier to debug in renderdoc when everything is labelled.